Bigtable/tests/System/FilterTest.php

```diff
@@ -0,0 +1,451 @@
+                [
+                    'rowKeys' => ['rk5'],
+                    'filter' => Filter::timestamp()->range()->of(3000, 5000)
+                ],
+                [
+                    'rk5' => [
+                        'cf1' => [
+                            'cq2' => $expectedRows['rk5']['cf1']['cq2'],
+                            'cq3' => $expectedRows['rk5']['cf1']['cq3']
+                        ]
+                    ]
+                ],
+                'testTimestampRange failed'
+            ],
+            [
+                [
+                    'rowKeys' => ['rk5'],
+                    'filter' => Filter::value()->exactMatch('value1')
+                ],
+                [
+                    'rk5' => [
+                        'cf1' => [
+                            'cq1' => $expectedRows['rk5']['cf1']['cq1']
+                        ]
+                    ]
+                ],
+                'testValueExactMatch failed'
+            ],
+            [
+                [
+                    'rowKeys' => ['rk5'],
+                    'filter' => Filter::value()->regex('value[12]+')
+                ],
+                [
+                    'rk5' => [
+                        'cf1' => [
+                            'cq1' => $expectedRows['rk5']['cf1']['cq1'],
+                            'cq2' => $expectedRows['rk5']['cf1']['cq2']
+                        ]
+                    ]
+                ],
+                'testValueRegex failed'
+            ],
+            [
+                [
+                    'rowKeys' => ['rk5'],
+                    'filter' => Filter::value()->range()->of('value1', 'value3')
+                ],
+                [
+                    'rk5' => [
+                        'cf1' => [
+                            'cq1' => $expectedRows['rk5']['cf1']['cq1'],
+                            'cq2' => $expectedRows['rk5']['cf1']['cq2']
+                        ]
+                    ]
+                ],
+                'testValueRange failed'
+            ],
+            [
+                [
+                    'rowKeys' => ['rk7'],
+                    'filter' => Filter::value()->strip()
+                ],
+                [
+                    'rk7' => [
+                        'cf1' => [
+                            'cq1' => [[
+                                'value' => '',
+                                'labels' => '',
+                                'timeStamp' => 2000
+                            ]]
+                        ]
+                    ]
+                ],
+                'testValueStrip failed'
+            ],
+        ];
+    }
+
+    public function testSample()
+    {
+        $rowFilter = Filter::key()->sample(.50);
+        $rows = iterator_to_array(
+            self::$dataClient->readRows(
+                [
+                    'filter' => $rowFilter
+                ]
+            )->readAll()
+        );
+        $this->assertTrue(count($rows) > 0);
```

Comment:
We could use [`assertGreaterThan`](https://phpunit.readthedocs.io/en/7.3/assertions.html#assertgreaterthan) here